<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewDialog</class>
 <widget class="QDialog" name="PreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreviewDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyEditorEBD.qrc">
    <normaloff>:/MyEditorEBD/Resources/preview.png</normaloff>:/MyEditorEBD/Resources/preview.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lEPath"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cBSort">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Files: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lBFileCount">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="FileDirTreeView" name="tVDir">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="DesignListWidget" name="lWImages"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileDirTreeView</class>
   <extends>QTreeView</extends>
   <header>FileDirTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>DesignListWidget</class>
   <extends>QListWidget</extends>
   <header>DesignListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyEditorEBD.qrc"/>
 </resources>
 <connections/>
</ui>
